src/plugins/vis_augmenter/server/saved_objects/augment_vis.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsType } from 'opensearch-dashboards/server';
+
+export const augmentVisSavedObjectType: SavedObjectsType = {
+  name: 'augment-vis',
+  hidden: false,
+  namespaceType: 'single',
+  management: {
+    // utilizing defaults for some of these fields
+    // icon: 'visualizeApp',
+    // defaultSearchField: 'id',
```

Comment:
so these commented lines are just to show the defaults?

Reply:
I'll remove these lines. Basically we can utilize the saved objects management plugin for the defaults on the icon and the search field, no need to specify/override here.

Reply:
cool - it's fine to leave them, I was just making sure I understood.